kerrgeopy/frequencies.py

```diff
@@ -4,9 +4,26 @@
 """
 from .constants import _standardize_params
 from .constants import *
-from scipy.special import ellipk, ellipe, elliprj, elliprf
+from scipy.special import ellipk, ellipe, elliprj, elliprf, elliprd
 from numpy import sin, cos, sqrt, pi, arcsin, floor, where
 
+def _ellipeinc(phi,m):
+    r"""
+    Incomplete elliptic integral of the second kind defined as :math:`E(\phi,m) = \int_0^{\phi} \sqrt{1-m\sin^2\theta}d\theta`.
+    """
+    # count the number of half periods
+    num_cycles = floor(phi/(pi/2))
+    # map phi to [0,pi/2]
+    phi = abs(arcsin(sin(phi)))
```

Comment:
A simpler method is to just take the modulus with respect to $\pi/2$
```
phi = abs(phi) % 0.5*np.pi
```

Reply:
@syp2001 pointed out that this does not quite work for half of the interval, so it is best to stick with his current implementation